pkg/provider/azure_loadbalancer.go

```diff
@@ -68,10 +68,10 @@ func (az *Cloud) existsPip(clusterName string, service *v1.Service) bool {
 		return existingPip
 	}
 
-	if v4Enabled && !existsPipSingleStack(false) {
+	if v4Enabled && !existsPipSingleStack(IsIPv4) {
```

Comment:
I think a better name would be IPVersionIPv4. IsIPv4 sounds like a variable, not a constant.

Reply:
Updated.